src/fips.c

```diff
@@ -51,6 +68,8 @@ static const char hmackey[] =
 
 int fips;
 
+#define LIBICA_FIPS_CONFIG		"/etc/libica/openssl3-fips.cnf"
```

Comment:
I would not hard-code `/etc` here.  That is usually configurable as `sysconfdir` and might default to `/usr/local/etc`.